.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
What is this? A comment in a JSON file?

Reply:
JSON5; that text comes from the stock VSCode generated file.